template/index.js

```diff
@@ -34,14 +34,12 @@ module.exports = yeoman.generators.NamedBase.extend({
         }
     },
 
-    writing: function files() {
+    files: function files() {
         debug("creating template '%s'", this.name);
         this.fs.copyTpl(
             this.templatePath('template.dust'),
             this.destinationPath(path.join('public', 'templates', this.name + '.dust')),
-            {
-                jsModule: this.options.jsModule
-            }
+            this.config.getAll()
```

Comment:
Seemed simpler, especially if someone makes changes to the templates in the future. It took me a while to even find that you had to explicitly pass along the arguments at all. I figured why wouldn't we just pass them all along?